cmd/dockerd/daemon.go

```diff
@@ -450,23 +450,21 @@ func newRouterOptions(ctx context.Context, config *config.Config, d *daemon.Daem
 			Named:    d.ImageNamedByBuildkit,
 		},
 	})
-	if err != nil {
-		return routerOptions{}, err
-	}
 
 	bb, err := buildbackend.NewBackend(d.ImageService(), manager, bk, d.EventsService)
 	if err != nil {
-		return routerOptions{}, errors.Wrap(err, "failed to create buildmanager")
+		return nil, nil, errors.Wrap(err, "failed to create buildmanager")
```

Comment:
As we're updating, perhaps we should change this to `failed to create build backend`, but see above; looks like this never returns an error.